dev-vcs/git-cola/git-cola-2.6.ebuild

```diff
@@ -87,7 +85,7 @@ python_install_all() {
 		prefix="${EPREFIX}/usr" \
 		install
 
-	python_fix_shebang "${ED}/usr/share/git-cola/bin/git-xbase"
+	python_fix_shebang "${ED}/usr/share/git-cola/bin/git-xbase" "${ED}"/usr/bin/git-cola
```

Comment:
Unneeded change.
See https://docs.python.org/2/distutils/setupscript.html#installing-scripts
and https://github.com/git-cola/git-cola/blob/v2.6/setup.py#L35